src/main/java/com/slack/kaldb/elasticsearchApi/searchRequest/SearchRequestSort.java

```diff
@@ -0,0 +1,15 @@
+package com.slack.kaldb.elasticsearchApi.searchRequest;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+public enum SearchRequestSort {
+  ASC,
+  DESC;
+
+  public static SearchRequestSort parse(JsonNode sort) {
+    if (sort != null) {
+      return SearchRequestSort.valueOf(sort.findValue("order").asText().toUpperCase());
+    }
+    return null;
```

Comment:
if sort is null should we return DESC always?

Reply:
Sure! That probably makes more sense.